toolchain/sem_ir/inst_fingerprinter.cpp

```diff
@@ -163,7 +163,8 @@ struct Worklist {
     }
   }
 
-  auto AddEntity(const EntityWithParamsBase& entity) -> void {
+  template <typename EntityT>
+  auto AddEntity(const EntityT& entity) -> void {
```

Comment:
Should we have a TODO to only instantiate this once for all types derived from EntityWithParamsBase?

Reply:
If we care, let's just fix it :) Done.